week1/homework/src/helper.js

```diff
@@ -0,0 +1,12 @@
+class Util {
+  static handleError(response) {
+    response.writeHead(404, { 'Content-Type': 'application/json' });
+    response.end(JSON.stringify({ error: 'Not found' }));
+  }
+  static sendFile(response, state) {
+    response.writeHead(200, { 'content-type': 'application/json' });
+    response.end(JSON.stringify(state));
```

Comment:
As line 8 this method is probably ending the response, or in other words it ends the process of the response and send it back to the client.

Straight to the point is this method have the right name of it's functionality? and `fileContent.js` file too?